.rubocop.yml

```diff
@@ -1,6 +1,7 @@
 inherit_from: ./.rubocop_todo.yml
 
 AllCops:
+  TargetRubyVersion: 2.1
```

Comment:
Is this the minimum ruby version?  If so, .travis.yml indicates it should be 2.0